src/Spd.Manager.Licence/ControllingMemberCrcAppContract.cs

```diff
@@ -20,6 +15,7 @@ public record ControllingMemberCrcAppBase
 {
     public WorkerLicenceTypeCode? WorkerLicenceTypeCode { get; set; }
     public ApplicationTypeCode? ApplicationTypeCode { get; set; }
+    public ApplicationOriginTypeCode ApplicationOriginTypeCode { get; set; } = ApplicationOriginTypeCode.Portal;
```

Comment:
we can set this directly in repository mappings also, then no need to modify contract